src/pretalx/submission/models/submission.py

```diff
@@ -124,6 +124,14 @@ class Submission(LogMixin, models.Model):
     notes = models.TextField(
         null=True, blank=True, verbose_name=_('Notes for the organiser')
     )
+    internal_notes = models.TextField(
+        null=True,
+        blank=True,
+        verbose_name=_('Notes by the organiser'),
+        help_text=_('Notes by the organiser for themselves (not visible for the speaker and the public)')
```

Comment:
Please add a '.' at the end of the sentence, as the follow-up remark on markdown looks weird otherwise.